tests/unit/TestPrefixConfig.js

```diff
@@ -0,0 +1,24 @@
+describe('AMO Prefix config', () => {
+  const existingNodeEnv = process.env.NODE_ENV;
+
+  afterEach(() => {
+    process.env.NODE_ENV = existingNodeEnv;
+    delete process.env.NODE_APP_INSTANCE;
+  });
+
+  it(`checks clientAppRoutes items are present in validClientAppUrlExceptions`, () => {
+    /*
+     * Without this the clientAppRoutes will not work as expected. If you want
+     * /en-US/blah to hit the app, 'blah' must be in both clientAppRoutes
+     * and validClientAppUrlExceptions in the app config.
+     */
+    process.env.NODE_ENV = 'production';
```

Comment:
😱 Seems scary. Is that just for this test? It's changed back afterward, right?

Reply:
Yeah - see the afterEach ^